api/client.go

```diff
@@ -30,20 +32,17 @@ type (
 		BaseURL string
 
 		// Services used for talking with different parts of the Recurly API
-		Accounts      *AccountsService
-		Adjustments   *AdjustmentsService
-		Billing       *BillingService
-		Coupons       *CouponsService
-		Redemptions   *RedemptionsService
-		Invoices      *InvoicesService
-		Plans         *PlansService
-		AddOns        *AddOnsService
-		Subscriptions *SubscriptionsService
-		Transactions  *TransactionsService
+		Accounts      recurly.AccountsService
```

Comment:
Can you move this `Client` struct, as well as the two functions (`newRequest` and `do`) into the root package, and have `NewClient` below return `*recurly.Client`? Youll need to be `NewRequest` and `Do` exported methods and update each of the services. Beyond that, I don't see anything that would cause import cycle issues in the `Client` or it's two functions.